kubernetes_asyncio/config/kube_config.py

```diff
@@ -34,7 +35,7 @@
 from .openid import OpenIDRequestor
 
 EXPIRY_SKEW_PREVENTION_DELAY = datetime.timedelta(minutes=5)
-KUBE_CONFIG_DEFAULT_LOCATION = os.environ.get('KUBECONFIG', '~/.kube/config')
+KUBE_CONFIG_DEFAULT_LOCATION = os.environ.get('KUBECONFIG', pathlib.Path.home() / '.kube/config')
```

Comment:
Thanks for your contribution, but as pathlib.Path.home() returns a PosixPath, converting to str is needed, could you fix it?

Reply:
Sorry for that, fixed in the [force push](https://github.com/tomplus/kubernetes_asyncio/commit/393c1e4f79346edd56bca70a4efdce96736992ce)